### PR TITLE
`aoi` definition in variables_style_rules.csv

### PR DESCRIPTION
**Pull request by RDaxini**
- [ ] ~Closes #xxxx~
 - [X] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [ ] ~Tests added~
 - [ ] ~Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/main/docs/sphinx/source/reference) for API changes.~
 - [ ] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [X] Pull request is nearly complete and ready for detailed review.
 - [ ] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

[Original definition](https://pvlib-python.readthedocs.io/en/stable/user_guide/variables_style_rules.html) says angles "between 90deg and 90deg", but this should be plus/minus 90deg (right?)

### CODE REVIEW

**Comment by cwhanse** [comment]
I read "angle...between" as going to tell me the two rays that define the angle, rather than the limits. What about "angle between the normal vector and the vector pointing to the sun's center"? I also think we can drop the limits +/-90 since pvlib doesn't enforce that.

**Comment by echedey-ls** [comment]
In addition to @cwhanse note, I would add `normal vector of the module`. To be very clear.

**Comment by kandersolar** [comment]
Just a general comment: AOI goes from 0 to 180.  Negative AOI doesn't make sense.

**Comment by RDaxini** [comment]
> Just a general comment: AOI goes from 0 to 180. Negative AOI doesn't make sense.

I was wondering about that...

I was trying to make sense of this line:
https://github.com/pvlib/pvlib-python/blob/cbe4cc5a8bb3bb529f2dd885aebcd527fedbbc63/pvlib/iam.py#L273

Maybe in this case, the normal to the surface is defined as the 0, and then +/- 90 either side?
As far as I can tell the only iam=0 is set in [line 228](https://github.com/pvlib/pvlib-python/blob/cbe4cc5a8bb3bb529f2dd885aebcd527fedbbc63/pvlib/iam.py#L228) for aoi>=90 (no mention of any negatives) so does this function docstring need revising?

Anyway, I'll remove the range reference as suggested by @cwhanse

**Comment by RDaxini** [comment]
> I read "angle...between" as going to tell me the two rays that define the angle, rather than the limits. 

Makes sense

> "the vector pointing to the sun's center"

Would "sun's rays" be sufficient here or is it not precise enough (I looked [here on sciencedirect](https://www.sciencedirect.com/topics/engineering/solar-incidence-angle#:~:text=INCIDENCE%20ANGLE%2C%20%CE%B8-,The%20solar%20incidence%20angle%2C%20%CE%B8%2C%20is%20the%20angle%20between%20the,%2C%20%CE%A6%2C%20are%20the%20same.) and [here on sandia](https://pvpmc.sandia.gov/modeling-guide/1-weather-design-inputs/plane-of-array-poa-irradiance/calculating-poa-irradiance/angle-of-incidence/) for lexical inspiration 👀)

**Comment by cwhanse** [comment]
@RDaxini that text probably originated in PVLib for Matlab, as a warning to the user when AOI<0 was input. When AOI<0, the Martin-Ruiz function would give IAM<0. At some point in transfer to pvlib-python or creating of iam.py from functions in pvsystem.py, the message was confused.

`iam.martin_ruiz` handles `aoi<0` by setting the IAM to zero.

To be clear, I am suggesting deleting the range from the description of AOI in the Glossary. I would correct, not delete, the statement that IAM= for AOI outside of 0 <= aoi <=90.

Let's not mix editing docstrings with creating a glossary in this PR, it won't ever get finished. If you can create an issue to accumulate docstring edits as a checklist, that would be very helpful. An entry in that checklist: this [line](https://github.com/pvlib/pvlib-python/blob/cbe4cc5a8bb3bb529f2dd885aebcd527fedbbc63/pvlib/iam.py#L295) is a cut/paste error and should be removed.

**Comment by RDaxini** [comment]
> To be clear, I am suggesting deleting the range from the description of AOI in the Glossary. I would correct, not delete, the statement in the `martin_ruiz` docstring that IAM= for AOI outside of 0 <= aoi <=90.

Understood, we are on the same page here

> Let's not mix editing docstrings with creating a glossary in this PR, it won't ever get finished.

Agreed
 
> It would be very helpful if you can create an issue to accumulate docstring edits as a checklist. An entry in that checklist: this [line](https://github.com/pvlib/pvlib-python/blob/cbe4cc5a8bb3bb529f2dd885aebcd527fedbbc63/pvlib/iam.py#L295) is a cut/paste error and should be removed.

Okay I will do this 👍🏾

**Comment by RDaxini** [comment]
> this [line](https://github.com/pvlib/pvlib-python/blob/cbe4cc5a8bb3bb529f2dd885aebcd527fedbbc63/pvlib/iam.py#L295) is a cut/paste error and should be removed.

@cwhanse is the same line on [381](https://github.com/pvlib/pvlib-python/blob/cbe4cc5a8bb3bb529f2dd885aebcd527fedbbc63/pvlib/iam.py#L381) and [471](https://github.com/pvlib/pvlib-python/blob/cbe4cc5a8bb3bb529f2dd885aebcd527fedbbc63/pvlib/iam.py#L471) correct?

File: pvlib/data/variables_style_rules.csv [hunk 1]
```diff
@@ -11,7 +11,7 @@ ghi;global horizontal irradiance
 ghi_extra;horizontal irradiance at top of atmosphere (extraterrestrial)
 gri;ground-reflected irradiance
 ape;average photon energy
-aoi;angle of incidence between :math:`90\deg` and :math:`90\deg`
+aoi;angle of incidence. Angle between the surface normal vector and the vector pointing towards the sun's centre
```
Comment (kandersolar) [inline pvlib/data/variables_style_rules.csv:14]:
```suggestion
aoi;angle of incidence. Angle between the surface normal vector and the vector pointing towards the sun's center
```

American English, please :)

Comment (RDaxini) [reply pvlib/data/variables_style_rules.csv:14]: I feel persecuted
Jks, done 😂

**Comment by RDaxini** [comment]
> > "the vector pointing to the sun's center"
> 
> Would "sun's rays" be sufficient here or is it not precise enough? (I looked [here on sciencedirect](https://www.sciencedirect.com/topics/engineering/solar-incidence-angle#:~:text=INCIDENCE%20ANGLE%2C%20%CE%B8-,The%20solar%20incidence%20angle%2C%20%CE%B8%2C%20is%20the%20angle%20between%20the,%2C%20%CE%A6%2C%20are%20the%20same.) and [here on sandia](https://pvpmc.sandia.gov/modeling-guide/1-weather-design-inputs/plane-of-array-poa-irradiance/calculating-poa-irradiance/angle-of-incidence/) for lexical inspiration 👀)

Any thoughts on this? Just an idea, not a definite suggestion.

**Comment by kandersolar** [comment]
The sun's rays originate from a disc with an angular diameter of a degree or so.  It is the center of this disc which defines `aoi`, in the sense that pvlib uses it.  

Seems like nobody objects to the definition as it currently stands in this PR, so I'll go ahead and merge it.  Thanks @RDaxini!

**Pull request merged**